packages/protocol/contracts/rewards/StakingRewards.sol

```diff
@@ -407,6 +460,19 @@ contract StakingRewards is ERC721PresetMinterPauserAutoIdUpgradeSafe, Reentrancy
     return leverageMultiplier;
   }
 
+  /// @notice Calculate the exchange rate that will be used to convert the original staked token amount to the
+  ///   `baseStakingToken()` amount
+  /// @param positionType Type of the staked postion
+  function getBaseTokenExchangeRate(StakedPositionType positionType) public view virtual returns (uint256) {
+    if (positionType == StakedPositionType.CurveLP) {
+      return config.getFiduUSDCCurveLP().getVirtualPrice() / config.getSeniorPool().sharePrice();
```

Comment:
* We need to handle units / flooring during the integer arithmetic e.g. multiply the numerator by the denominator's mantissa. Otherwise, the result will only be a whole number (i.e. 0, 1, or 2 in extreme cases)
* We should use SafeMath's `.div` instead of `/`

Reply:
Updated